src/components/Nav/Nav.jsx

```diff
@@ -1,6 +1,175 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { withRouter } from 'react-router-dom';
+
+import SiteMap from './SiteMap';
+import SearchBar from './SearchBar';
 
 function Nav() {
-  return <div></div>;
+  const [isSiteMapClicked, setIsSiteMapClicked] = useState(false);
+  const [isSearchClicked, setIsSearchClicked] = useState(false);
+
+  const onSiteMapClick = () => {
+    if (isSearchClicked) setIsSearchClicked(!isSearchClicked);
+    setIsSiteMapClicked(!isSiteMapClicked);
+  };
+
+  const onSearchClick = () => {
+    if (isSiteMapClicked) setIsSiteMapClicked(!isSiteMapClicked);
+    setIsSearchClicked(!isSearchClicked);
+  };
+
+  const Navigator = styled.nav`
+    position: relative;
+    overflow: hidden;
+    background-color: rgba(0, 0, 0, 0.3);
+    top: 0;
+    width: 100%;
+    height: 91px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+  `;
+
+  const UpperWrapper = styled.div`
+    width: 1100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  `;
+
+  const LowerWrapper = styled.div`
+    width: 1100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  `;
+
+  const UpperLeftLink = styled.a`
```

Comment:
SPA 앱에서는 a 태그를 사용하면 기본적으로 브라우저에서 새로고침 이벤트가 일어납니다. 그러면 앱에서 원래 가지고 있던 컨텍스트를 잃어버리기 때문에 a 태그 대신 react-router-dom의 Link 컴포넌트를 사용해주세요
```suggestion
  const UpperLeftLink = styled(Link)`
```

Reply:
넵. a태그는 Link로 바꾸겠습니다.